### ISSUE TITLE
Can't Click on Links for Age-Restricted Trailers

### ISSUE THREAD

**Issue by Hossy** [petio-team/petio#252]
**Describe the bug**
When trying to play a trailer (e.g. Mortal Kombat 2021), I get the following message but cannot click any of the links.
![image](https://user-images.githubusercontent.com/1756846/111845790-8d000d80-88d3-11eb-8c6c-2b6478f601e3.png)

**To Reproduce**
1. Go to Mortal Kombat 2021
2. Click on Trailer
3. Try to click

**Expected behavior**
I would be able to click the link to open YouTube to watch the trailer there after, presumably, they make me login or somehow verify my age.

**Screenshots**
See description

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 88.0.4324.190

**Additional context**
Alternatively, upon trying to play the trailer, detect the video not playing and inform the user that is isn't Petio's fault.  Maybe offer a link to tmdb/tvdb/search engine to attempt to find a trailer there.

**Comment by RyleaStark** [comment]
This would require the GoogleAPI for YouTube to detect `contentDetails.regionRestriction`, `contentDetails.contentRating`, `contentDetails.licensedContent`, or `status.embeddable`.

The API is free but is limited to 10k requests per day, so we would probably need a global service for Petio Clients to connect to and gather this information as part of discovery with tmdb; However I could be overcomplicating the thought here.

**Comment by AshDyson** [comment]
Yeah I think we can implement a pre-lookup on the trailer play and handle it differently
